readme.md

```diff
@@ -86,6 +86,22 @@ Run `np` without arguments to launch the interactive UI that guides you through
 <img src="screenshot-ui.png" width="1290">
 
 
+## Config
+
+`np` can be configured both locally and globally. When using the global `np` binary, you can configure any of the CLI flags in either a `.np-config.js` or `.np-config.json` file in the home directory. When using the local `np` binary, for example, in a `npm run` script, you can configure `np` by setting the flags in either a top-level `np` field in `package.json` or in a `.np-config.js` or `.np-config.json` file in the project directory.
+
```

Comment:
We should mention that the flags are to be used in a camelCased form.

Reply:
Maybe we should just list them explicitly? There are not that many.

Reply:
For now, I guess we can just list them explicitly